colour/colorimetry/tests/test_tristimulus_values.py

```diff
@@ -1320,6 +1322,91 @@ def test_raise_exception_sd_to_XYZ_ASTME308(self):
         )
 
 
+class Test_Absolute_spd_to_XYZ(unittest.TestCase):
```

Comment:
**Minor**

Needs to be added to `__all__`, and I would probably move it after the main test classes.